src/main/java/com/codeborne/selenide/appium/WebdriverUnwrapper.java

```diff
@@ -13,6 +14,10 @@
  * Will be replaced by a better solution in Selenide.
  */
 public class WebdriverUnwrapper {
+  public static boolean isMobile(Driver driver) {
```

Comment:
would it be better to move it to `AppiumDriverRunner`?

Reply:
@BorisOsipov  No, `AppiumDriverRunner` holds a ThreadLocal instance of webdriver.
But in this case we get `Driver` instance from `WebElementSource.driver()` in `Command` subclasses - which is not static/threadlocal.

Reply:
Seems legit. But I don't get how `WebdriverUnwrapper` is related to check if current webdriver mobile or not too.

Reply:
because it uses method `instanceOf` -> `cast` from the same class which basically "unwraps" the webdriver.